java/src/main/native/ai_onnxruntime_OrtEnvironment.c

```diff
@@ -24,6 +25,27 @@ JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OrtEnvironment_createHandle(JNIEnv *
     return (jlong) env;
 }
 
+/*
+ * Class:     ai_onnxruntime_OrtEnvironment
+ * Method:    createHandle
+ * Signature: (JILjava/lang/String;J)J
+ */
+JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OrtEnvironment_createHandle__JILjava_lang_String_2J
+  (JNIEnv * jniEnv, jclass jobj, jlong apiHandle, jint loggingLevel, jstring name, jlong threadOptionsHandle) {
+    (void) jobj; // Required JNI parameter not needed by functions which don't need to access their host object.
```

Comment:
> jobj [](start = 10, length = 5)

What if you just commented out the param name?

Reply:
The function signature is generated by JNI, I can't control it. And the compiler errors if there are unused arguments.

Reply:
Oh wait, I see what you mean. I'll check.